app/controllers/admin/profiles/admin_profiles_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Admin
+  module Profiles
+    class AdminProfilesController < Admin::BaseController
+      skip_after_action :verify_policy_scoped, only: %i[show destroy]
+      skip_after_action :verify_authorized
+
+      def index
+        authorize AdminProfile, :index?
```

Comment:
`:index?` is not needed here. You only need to specify when it doesn't match the action name

Reply:
Done